### PR TITLE
Voice Assistant: 13 USD task: make some components reusable

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Voice Assistant: 13 USD task: make some components reusable
- Make wake word installation and firmware installation sections reusable
- to prepare for S3-Box task

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/voice_assistant/add_wake_word_to_voice_assistant.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Adding a wake word to your voice assistant
```
Comment (kbx81) [inline source/_includes/voice_assistant/add_wake_word_to_voice_assistant.md:1]: Usually for headings I recommend Title Case:
```suggestion
## Adding a Wake Word to your Voice Assistant
```

Comment (kbx81) [reply source/_includes/voice_assistant/add_wake_word_to_voice_assistant.md:1]: (Same for other headings below)

**Review by kbx81 — APPROVED** [review]
One quick nit about headings, otherwise LGTM

File: source/_includes/voice_assistant/add_wake_word_to_voice_assistant.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## Adding a Wake Word to your Voice Assistant
```
Comment (c0ffeeca7) [inline source/_includes/voice_assistant/add_wake_word_to_voice_assistant.md]:
```suggestion
## Adding a wake word to your voice assistant
```

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**